libs/fixtures/.gitignore

```diff
@@ -1,2 +1,3 @@
 coverage
 node_modules
+dist
```

Comment:
Nothing should be using `dist` anymore. It's `build`. This is probably here because you saw it's no longer ignored by git and you saw them as new untracked files? Just `rm -rf dist`.

Reply:
Ahh I thought I had accidently did pnpm build in a weird place and created them. Will remove.